product_to_bizerba_scale/models/product_template.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2014-2017 GRAP (http://www.grap.coop)
+#   - Sylvain LE GAL (https://twitter.com/legalsylvain)
+# Copyright 2017-Today Coop IT Easy SCRLfs
+#   - Houssine BAKKALI <houssine@coopiteasy.be>
+#   - Rémy TAYMANS <remy@coopiteasy.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models, _
+from odoo.addons import decimal_precision as dp
+
+# TODO: scale_category is defined in beesdoo_product but this module do
+#       not depend on it. Find a way to configure these fields.
```

Comment:
Ok, for now,  add `beesdoo_product` in the dependencies and leave this comment here. :)

Reply:
replaced `product` with `beesdoo_product`. Should we move this module to `beescoop/obeesdoo`?